docs/tutorials/index.rst

```diff
@@ -20,11 +35,26 @@ port your existing code to the Determined environment:
 |                                 | port a ``tf.keras`` model to Determined.                     |
 +---------------------------------+--------------------------------------------------------------+
 
+Looking for Examples?
+=====================
+
+Visit our `Examples <https://docs.determined.ai/latest/example-solutions/examples.html>`_ page then
+open the ``examples/`` subdirectory of the `Determined GitHub repo
+<https://github.com/determined-ai/determined/tree/master/examples>`__.
+
+Go Further
```

Comment:
We can maybe remove "Go Further". By the time the reader gets to this phrase, they already have the Examples page open with further steps.
As it is now, it looks like there's something missing.

Reply:
makes sense, switched them around